### PR TITLE
 164678845  database setup

### PR DESCRIPTION
**Pull request by mnzube**
#### What does this PR do?
creates the database 
#### Description of Task to be completed?
installed packages pg and dotenv
#### How should this be manually tested?
view package.json file and new file  .env
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164678845
#### Questions:

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://postgres:12345@localhost:5433/Epic-Mail
```
Comment (fob413) [inline .env]: your `.env` is still online. Please remove this file from github

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,8 @@
 node_modules/
-.env
 .nyc_output
 .vscode/
 coverage
-
+.gitignore
```
Comment (fob413) [inline .gitignore]: You should not `.gitignore`, `.gitignore`

File: .env (hunk 1 above)
Comment (mnzube) [reply .env]: removed it from my branch

File: .gitignore (hunk 2 above)
Comment (mnzube) [reply .gitignore]: removed the. gitignore file

File: SERVER/db/tables.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import {Pool}from "pg";
+import dotenv from "dotenv";
+dotenv.config();
+
+const pool=new Pool({
+    connectionString:process.env.DATABASE_URL
+});
+pool.on('connect',()=>{
```
Comment (fob413) [inline SERVER/db/tables.js]: Why not import this pool connection from your `db.js` file you already created.

**Review by fob413 — CHANGES_REQUESTED** [review]
Please address the comment

File: SERVER/db/tables.js (hunk 3 above)
Comment (mnzube) [reply SERVER/db/tables.js]: done

**Pull request merged**